tests/python/ci/test_ci.py

```diff
@@ -342,9 +341,12 @@ def run(files, should_skip):
 
 
 def test_ping_reviewers(tmpdir_factory):
+    """
+    Test that reviewers are messaged after a time period of inactivity
+    """
     reviewers_script = REPO_ROOT / "tests" / "scripts" / "ping_reviewers.py"
 
-    def run(pr, check):
+    def run(pr, check):  # pylint: disable=invalid-name
```

Comment:
Wouldn't it be better to fix the function name than ignore the linter warning?

Reply:
the issue is on `pr` (says its too short), but that is what it is and I don't think the linter is correct here

Reply:
I don't see a reason to not use `pull_request` here which is more immediately obvious to the reader?